aiida/cmdline/commands/cmd_data/cmd_show.py

```diff
@@ -10,6 +10,9 @@
 """
 This allows to manage showfunctionality to all data types.
 """
+import sys
```

Comment:
Could those have been put in the functions to improve startup performance?

Reply:
done!

you're right, here it makes sense to move imports inside functions (although those system libraries are likely imported somewhere anyhow)